test/format/config.c

```diff
@@ -760,6 +757,23 @@ config_in_memory_reset(void)
     }
 }
 
+/*
+ * config_backup_incr_compatibility_check --
+ *     Backup incremental compatibility check.
+ */
+static void
+config_backup_incr_compatibility_check(void)
```

Comment:
How about specifying the `log` in the function name and description? This function deals with incremental=log.

Reply:
Updated the function name and the function description.